cmd/prometheus/main.go

```diff
@@ -97,6 +98,11 @@ var (
 
 	agentMode                       bool
 	agentOnlyFlags, serverOnlyFlags []string
+
+	defaultGoCollectors = collectors.WithGoCollectorRuntimeMetrics(
```

Comment:
I'm unclear what is meant by "default" here.  There is no way to change them; it's just the collectors that we picked, right?

Reply:
Yea, `default` was just that this is the Prometheus default. How about just `goCollectors`?

Reply:
I eliminated the var altogether.